### PR TITLE
feat: add open file button to the title of diff editor

### PR DESCRIPTION
**Pull request by conwnet**

### CODE REVIEW

File: extensions/github1s/package.json [hunk 1]
```diff
@@ -90,6 +91,12 @@
         "command": "github1s.open-gitpod",
         "title": "Edit files in Gitpod",
         "category": "GitHub1s"
+      },
+      {
+        "command": "github1s.open-diff-head-file",
+        "title": "Open File",
+        "category": "GitHub1s",
+        "icon": "$(go-to-file)"
```
Comment (xcv58) [inline extensions/github1s/package.json:99]: Do we need to register the comment here?
image
The command will be shown on this view and it makes no action regardless it's in diff view or not.

It only works when I click this icon:
image

Comment (conwnet) [reply extensions/github1s/package.json]: Thank you so much for pointing out the problem!

we have to put the command to `contributes.commands` to indicate the `icon` and the `title`.

I found we can hide it from commandPalette by editing `contributes.menus.commandPalette` in `package.json`

**Comment by conwnet** [comment]
image

**Review by xcv58 — APPROVED** [review]
LGTM!

**Pull request merged**